src/server.test.ts

```diff
@@ -51,9 +51,9 @@ test('find "Ehrenbergstr."', () => {
     })
 });
 
-test('find "Kasseler Str"', () => {
-    const { addresses, count } = autoCompleteAddress('kasseler');
-    expect(addresses.length).toEqual(1);
+test('find "Kasseler Str"', async () => {
+    const { addresses, count } = await autoCompleteAddress('kasseler');
+    expect(addresses.length).toEqual(2);
```

Comment:
Ich glaube hier müsste 2 stehen, weil es zwei Adresse sind, die gefunden werden

Reply:
YOU ACTIVATED MY TRAP CARD